src/components/Feed/ProposalCreator.js

```diff
@@ -1,4 +1,5 @@
 import React, { useEffect, useState } from 'react'
+import { Link } from 'react-router-dom'
```

Comment:
As discussed on Discord, let´s use the `Link` component from `@1hive/1hive-ui`.

Reply:
You an see an example [here](https://github.com/1Hive/honey-pot/blob/bd098f0df7c56360e413d8a3c2a367a588c5f918/src/components/Profile/Activity.js?_pjax=%23js-repo-pjax-container%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%2C%20%5Bdata-pjax-container%5D#L88)